packages/@glimmer/runtime/lib/references.ts

```diff
@@ -1,5 +1,6 @@
 import { Option, Recast } from '@glimmer/interfaces';
 import { ConstReference, PathReference, Reference, CachedReference } from '@glimmer/reference';
+import { toBool as defaultToBool } from '@glimmer/global-context';
```

Comment:
Why can't we call this `toBool` (e.g. not alias)? Seems like the only consumer that is trying to customize is the curried component definition (which uses a custom subclass), could we update `ConditionalReference` to not take a second constructor arg and have the subclass assign `this.toBool` itself?

Reply:
There is one `ConditionalReference` usage that currently uses a custom `toBool`, this will be fixed in the Monomorphic References PR (since they'll both just become compute refs), so I decided to keep it as is for the time being.